config/allowlist.go

```diff
@@ -88,6 +109,80 @@ func (a *Allowlist) Validate() error {
 		a.StopWords = maps.Keys(uniqueStopwords)
 	}
 
+	// Compile the expression
+	if useExpression && a.Expression != "" {
+		// Build the environment: variables and functions available to the users.
+		// TODO: Is it safe to reuse this across multiple expressions?
```

Comment:
I would hope so 🤔

Reply:
i.e., can `env` be declared as a top-level variable instead of being declared for each whitelist?

Reply:
yep, correctomundo